modules/gammaBidAdapter.js

```diff
@@ -0,0 +1,85 @@
+import * as utils from 'src/utils';
+import { registerBidder } from 'src/adapters/bidderFactory';
+import find from 'core-js/library/fn/array/find';
+
+const BIDDER_CODE = 'gamma';
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['gamma'],
+
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {object} bid The bid to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: function(bid) {
+    return !!(bid.params.siteId || bid.params.zoneId || bid.params.gaxDomain);
```

Comment:
Is the `gaxDomain` something you want to always specify within the params object?  Is this value going to change often?  Other adapters typically setup their own variable for this type of data point in the adapter file so that it's just innately available, so wanted to confirm here.

Reply:
The gaxDomain is endpoint domain, it's required and we do not change often.

Reply:
Hi @gammassp, 

In that case - wouldn't it be more ideal/convenient to have the endpoint incorporated into your adapter's code directly rather than rely on it be passed from the `param` object?  It would be one less thing to maintain/verify within your code, as you would always know what the value would be.

Reply:
Hi @jsnellbaker,

The gaxDomain param was removed and moved into adapter, please review again.

Thanks so much